ct.yaml

```diff
@@ -0,0 +1,8 @@
+# See https://github.com/helm/chart-testing#configuration
+remote: origin
+target-branch: master
+chart-dirs:
+  - helm-charts
```

Comment:
I think that this is supposed to be `helm-chart`. When I set that, `ct` complains about the directory not being a valid chart directory:

> Directory 'helm-chart/templates' is not a valid chart directory. Skipping...

Reply:
Good catch! :+1:

Reply:
This still doesn't work for me. I also tried running it in a docker container. Am I missing something?